HyperV/scripts/create-environment.ps1

```diff
@@ -306,6 +363,50 @@ Copy-Item -Recurse $configDir "$remoteConfigs\$hostname"
 
 Write-Host "Starting the services"
 
+
+
+if ($jobType -eq 'smbfs')
+{
+    $currDate = (Get-Date).ToString()
+    Write-Host "$currDate Starting cinder-volume service"
+    Try
+    {
+        Start-Service cinder-volume
+    }
+    Catch
+    {
+        Write-Host "Can not start the cinder-volume service."
+    }
+    Start-Sleep -s 30
+    if ($(get-service cinder-volume).Status -eq "Stopped")
+    {
+        Write-Host "cinder-volume service is not running."
+        $currDate = (Get-Date).ToString()

```

Comment:
I think that in a separate patch, we should replace all the occurrences of 'write-host' to 'log_message', as it already logs the timestamp and it will allow us to easily add other information as well.